.werft/vm/manifests.ts

```diff
@@ -0,0 +1,149 @@
+type NamespaceManifestOptions = {
+  namespace: string
+}
+
+export function NamespaceManifest({ namespace }: NamespaceManifestOptions) {
+  return `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${namespace}
+`
+}
+
+type VirtualMachineManifestArguments = {
+  vmName: string
+  namespace: string
+  claimName: string,
+  userDataSecretName: string
+}
+
+export function VirtualMachineManifest({ vmName, namespace, claimName, userDataSecretName }: VirtualMachineManifestArguments) {
+  return `
+apiVersion: kubevirt.io/v1
+type: kubevirt.io.virtualmachine
+kind: VirtualMachine
+metadata:
+  namespace: ${namespace}
+  annotations:
+    harvesterhci.io/volumeClaimTemplates: '[{"metadata":{"name":"${claimName}","annotations":{"harvesterhci.io/imageId":"default/image-cjlm2"}},"spec":{"accessModes":["ReadWriteMany"],"resources":{"requests":{"storage":"10Gi"}},"volumeMode":"Block","storageClassName":"longhorn-image-cjlm2"}}]'
+    network.harvesterhci.io/ips: "[]"
+  labels:
+    harvesterhci.io/creator: harvester
+    harvesterhci.io/os: ubuntu
+  name: ${vmName}
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        harvesterhci.io/sshNames: "[]"
+      labels:
+        harvesterhci.io/vmName: ${vmName}
+    spec:
+      domain:
+        hostname: ${vmName}
+        machine:
+          type: q35
```

Comment:
Can we use native virtualization instead of Qemu?

Reply:
@meysholdt It's not clear to me which machines types are considered "native". I was assuming that Qemu would use hardware virtulization either way when available. Their docs are a [bit sparse](https://kubevirt.io/user-guide/virtual_machines/virtual_hardware/#machine-type) but I'm happy to try any type that you'd like if you have a suggestion ☺️

Reply:
as discussed offline, q35 seems fine and seems to do hardwave acceleration.